modules/fluid_properties/src/userobjects/NaKFluidProperties.C

```diff
@@ -0,0 +1,270 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NaKFluidProperties.h"
+
+registerMooseObject("FluidPropertiesApp", NaKFluidProperties);
+
+InputParameters
+NaKFluidProperties::validParams()
+{
+  InputParameters params = SinglePhaseFluidProperties::validParams();
+  params.addClassDescription("Fluid properties for NaK");
+  params.addRequiredParam<Real>(
+      "weight_fraction_K",
+      "Weight fraction of potassium in NaK. Only eutectic is implemented (X_K = 0.778)");
```

Comment:
this seems unnecessary but it matters a lot so I want people to realize that when they make their input file

Reply:
Would you just want to name this as `EutecticNaKFluidProperties` and then not have this parameter that will likely force users to get an error at least once? It's counter-intuitive to require an input that can only be one thing

Reply:
it s required though so they will specify it

I like NaK because it goes next to NaCl in the index. Prefer eutectic?

Reply:
I do, but maybe another reviewer will feel otherwise. @joshuahansel ?

Reply:
@tanoret ?

Reply:
> if they know what the eutectic is

That and know to specify it to exactly three significant figures, so yea they would likely have to read the parameter description first ... or just copy it from example input files which is probably what they will be doing. I suppose that this is ok